hbw/app/javascript/packs/hbw/widget.js.jsx

```diff
@@ -61,13 +56,6 @@ modulejs.define(
 
       changeTask (task) {
```

Comment:
https://github.com/latera/homs/pull/391/files#diff-2e271f00419ac5da37c3b151a10c6994L58
Are you sure it's still used?

Reply:
Link does not work.
But is used: 
```
this.env.dispatcher.bind('hbw:task-clicked', 'widget', this.changeTask);
```
And it is triggering in `entity_task/s.js.jsx`.

Reply:
Yes, link navigated to this line :)
I see that it's triggering but I wonder why. I thought these events were about side menu only. Am I wrong?